lib/Driver/ToolChains.cpp

```diff
@@ -1191,6 +1225,20 @@ addLinkSanitizerLibArgsForDarwin(const ArgList &Args,
       /*AddRPath=*/ shared, TC);
 }
 
+static void
+addLinkSanitizerLibArgsForWindows(const ArgList &Args,
+                                 ArgStringList &Arguments,
+                                 StringRef Sanitizer,
+                                 const ToolChain &TC,
+                                 bool shared = true
```

Comment:
Nitpick: misaligned arguments here, and the close paren should follow the last argument instead of going on the next line.